sklearn/metrics/_classification.py

```diff
@@ -685,6 +685,11 @@ def jaccard_score(y_true, y_pred, *, labels=None, pos_label=1,
     sample_weight : array-like of shape (n_samples,), default=None
         Sample weights.
 
+    zero_division : "warn", 0 or 1, default="warn"
+        Sets the value to return when there is a zero division, i.e. when all
+        predictions and labels are negative. If set to "warn", this acts as 0,
```

Comment:
In this context, there should be no negative values in the predictions and labels?

Reply:
Updated doc string.